xml/nmra_mfg_list.xml

```diff
@@ -1,212 +1,217 @@
-  <manufacturer mfg="QElectronics GmbH" mfgID="55" />
-  <manufacturer mfg="QSIndustries" mfgID="113" />
-  <manufacturer mfg="RR-CirKits" mfgID="87" />
-  <manufacturer mfg="Railflyer Model Prototypes, Inc." mfgID="84" />
-  <manufacturer mfg="Railnet Solutions, LLC" mfgID="66" />
-  <manufacturer mfg="Rails Europ Express" mfgID="146" />
-  <manufacturer mfg="Railstars Limited" mfgID="91" />
-  <manufacturer mfg="RamFixx Technologies (Wangrow)" mfgID="15" />
-  <manufacturer mfg="Rampino Elektronik" mfgID="57" />
-  <manufacturer mfg="Rautenhaus Digital Vertrieb" mfgID="53" />
-  <manufacturer mfg="RealRail Effects" mfgID="139" />
-  <manufacturer mfg="Regal Way Co. Ltd" mfgID="32" />
-  <manufacturer mfg="Rock Junction Controls" mfgID="149" />
-  <manufacturer mfg="Roco Modelspielwaren" mfgID="161" />
-  <manufacturer mfg="Rocrail" mfgID="70" />
-  <manufacturer mfg="S Helper Service" mfgID="23" />
-  <manufacturer mfg="SLOMO Railroad Models" mfgID="142" />
-  <manufacturer mfg="SPROG DCC" mfgID="44" />
-  <manufacturer mfg="Sanda Kan Industrial" mfgID="95" />
-  <manufacturer mfg="Shourt Line" mfgID="90" />
-  <manufacturer mfg="Signaling Solution" mfgID="119" />
-  <manufacturer mfg="Silicon Railway" mfgID="33" />
-  <manufacturer mfg="SoundTraxx (Throttle-Up)" mfgID="141" />
-  <manufacturer mfg="Spectrum Engineering" mfgID="80" />
-  <manufacturer mfg="T4T - Technology for Trains GmbH" mfgID="20" />
-  <manufacturer mfg="TCH Technology" mfgID="54" />
-  <manufacturer mfg="Tam Valley Depot" mfgID="59" />
-  <manufacturer mfg="Tams Elektronik GmbH" mfgID="62" />
-  <manufacturer mfg="Tawcrafts" mfgID="92" />
-  <manufacturer mfg="Team Digital, LLC" mfgID="25" />
-  <manufacturer mfg="Tehnologistic (train-O-matic)" mfgID="78" />
-  <manufacturer mfg="The Electric Railroad Company" mfgID="73" />
-  <manufacturer mfg="Throttle-Up (SoundTraxx)" mfgID="141" />
-  <manufacturer mfg="Train Control Systems" mfgID="153" />
-  <manufacturer mfg="Train ID Systems" mfgID="138" />
-  <manufacturer mfg="Train Technology" mfgID="2" />
-  <manufacturer mfg="TrainModules" mfgID="61" />
-  <manufacturer mfg="TrainTech" mfgID="104" />
-  <manufacturer mfg="Trenes Digitales" mfgID="100" />
-  <manufacturer mfg="Trix Modelleisenbahn" mfgID="131" />
-  <manufacturer mfg="Uhlenbrock Elektronik" mfgID="85" />
-  <manufacturer mfg="Umelec Ing Buero" mfgID="147" />
-  <manufacturer mfg="Viessmann Modellspielwaren GmbH" mfgID="109" />
-  <manufacturer mfg="W.S. Ataras Engineering" mfgID="119" />
-  <manufacturer mfg="WP Railshops" mfgID="163" />
-  <manufacturer mfg="Wangrow" mfgID="12" />
-  <manufacturer mfg="Wekomm Engineering, GmbH" mfgID="86" />
-  <manufacturer mfg="Wm. K. Walthers, Inc" mfgID="150" />
-  <manufacturer mfg="ZTC" mfgID="132" />
-  <manufacturer mfg="Zimo" mfgID="145" />
-  <manufacturer mfg="csikos-muhely" mfgID="120" />
-  <manufacturer mfg="drM" mfgID="164" />
+      <manufacturer mfg="NMRA" mfgID="999" />
```

Comment:
Is it intended to change indentation from two spaces to six spaces? I'm not against it, except that it makes it difficult to see what's actually changed.

Reply:
True, but it matches the automated output in the decoderIndex file, which I have to compare it to occasionally.  `git diff -w` ignores whitespace. There's also that option in GitHub:
![image](https://user-images.githubusercontent.com/2774117/214939137-f0d67501-b788-46ce-acf7-a8bfbfad7077.png)

Reply:
Thanks!